packages/widgets/src/menubar.ts

```diff
@@ -387,6 +413,13 @@ export class MenuBar extends Widget {
     }
   }
 
+  /**
+   * A message handler invoked on a `'resize'` message.
+   */
+  protected onResize(msg: Widget.ResizeMessage): void {
+    this.update();
```

Comment:
You should propagate the message.

```suggestion
    this.update();
    super.onResize(msg);
```